### DISCUSSION TITLE
Why should I avoid 1password?

### DISCUSSION THREAD

**Discussion by izkluxcvy** [pluja/awesome-privacy#285] [Discussion]
According to [PrivacyGuides.org](https://www.privacyguides.org/passwords/#1password), 1password is a closed-source but certainly trustworthy service. Even if you are an open source believer, it should not be "Avoid".
Is PrivacyGuides wrong?

**Comment by spekulatius** [comment]
Jumping in with some thoughts: Privacy can't live without control of how the information is handled. Open source is a step in this direction giving you the chance to scrutinize and -most importantly- self-host. The only chance to get some form of privacy is to self-host. So I would tend to agree that closed-source is inherently less trustworthy.

**Comment by izkluxcvy** [comment]
Yes, Closed source is less reliable than open source. Self-hosting is a win/win situation where you can trust your own management or trust the provider, but it would certainly be nice to have a choice. But I don't think it should be avoided because of that; I think 1password has technical specifications worth trusting, as well as audits of them.

**Reply by pluja** [reply]
You are right. I removed 1password from avoid. But due to the reasons mentioned by @spekulatius it will not be recommended.

**Reply by spekulatius** [reply]
No doubt there is a gradient to privacy (and trust into services). I think we all appreciate this :)
